src/Control/Email/Email.php

```diff
@@ -706,30 +719,39 @@ public function setBody($body)
     }
 
     /**
+     * @deprecated 4.12.0 Will be replaced with Symfony\Component\Mime\Email::html()
```

Comment:
```suggestion
     * @deprecated 4.12.0 Use html() instead
```

Reply:
No making this change because `html()` doesn't exist, so this message makes no sense in 4.12

Reply:
Still need to remove the FQCN - the method will still be on _this_ class.

Reply:
Updated